### PR TITLE
Add Meitrack WiFi Aps, celltowers and network info

### LINKED ISSUE #5171: Meitrack: Support for WiFi Aps
**Is your feature request related to a problem? Please describe.**
Traccar Meitrack decoder does not support WiFi APs.

**Describe the solution you'd like**
An extension of the Meitrack decoder which can parse WiFi APs as well.

**Describe alternatives you've considered**
None, but I am implementing a fix at this moment.

**Additional context**
Issue based on [this forum post](https://www.traccar.org/forums/topic/extending-meitrack-protocol-to-parse-wifi-aps/).

### PR DESCRIPTION
**Pull request by berghdavid**
Includes WiFi aps, celltowers and network information.

Based on page 8-10 of Meitrack protocol:
https://www.meitrack.com/cd-download/Protocols/MEITRACK_P88L_GPRS_Protocol.pdf

Closes #5171

### CODE REVIEW

File: src/main/java/org/traccar/protocol/MeitrackProtocolDecoder.java [hunk 1]
```diff
@@ -537,6 +541,32 @@ private List<Position> decodeBinaryE(Channel channel, SocketAddress remoteAddres
...
+                    case 0x1D:
+                    case 0x1E:
+                    case 0x1F:
+                    case 0x20:
+                    case 0x21:
+                    case 0x22:
+                    case 0x23:
+                    case 0x24:
+                    case 0x25:
+                        String wifiMAC = ByteBufUtil.hexDump(buf.readSlice(6));
```
Comment (tananaev) [inline src/main/java/org/traccar/protocol/MeitrackProtocolDecoder.java:553]: Wifi MAC should be in the following format:
```
xx:xx:xx:xx:xx:xx
```

File: src/main/java/org/traccar/protocol/MeitrackProtocolDecoder.java [hunk 2]
```diff
@@ -548,6 +578,13 @@ private List<Position> decodeBinaryE(Channel channel, SocketAddress remoteAddres
                         buf.readUnsignedByte(); // label
                         position.set(Position.PREFIX_TEMP + (id - 0x2A), buf.readShortLE() * 0.01);
                         break;
+                    case 0x4B:
+                        position.set("networkVersion", buf.readUnsignedByte());
+                        position.set("networkType", buf.readUnsignedByte());
+                        int networkDescLen = buf.readUnsignedByte();
+                        String networkDesc = buf.readSlice(networkDescLen).toString(StandardCharsets.US_ASCII);
+                        network.setRadioType(networkDesc);
```
Comment (tananaev) [inline src/main/java/org/traccar/protocol/MeitrackProtocolDecoder.java:586]: Again, inline.

Also, what values does it return? It doesn't seem to match the expected types.

Comment (tananaev) [reply src/main/java/org/traccar/protocol/MeitrackProtocolDecoder.java]: If you want to use here, we need to know all possible values.

Comment (berghdavid) [reply src/main/java/org/traccar/protocol/MeitrackProtocolDecoder.java]: Ok, I'll ignore these parameters until I find more specifics about them.

File: src/main/java/org/traccar/protocol/MeitrackProtocolDecoder.java [hunk 3]
```diff
@@ -537,6 +541,27 @@ private List<Position> decodeBinaryE(Channel channel, SocketAddress remoteAddres
...
+                    case 0x1E:
+                    case 0x1F:
+                    case 0x20:
+                    case 0x21:
+                    case 0x22:
+                    case 0x23:
+                    case 0x24:
+                    case 0x25:
+                        String wifiMAC = ByteBufUtil.hexDump(buf.readSlice(6)).replaceAll("(..)", "$1:");
+                        int wifiRSSI = buf.readShortLE();
```
Comment (tananaev) [inline src/main/java/org/traccar/protocol/MeitrackProtocolDecoder.java:554]: 1. I think with your code you will have trailing colon after the address.
2. Variables are not following our naming conventions. Once again I recommend inlining them. And if you do need, they should lower camel case. For example `wifiMac`.

**Comment by 0xenca** [comment]
This should not be pulled until test passed

**Comment by tananaev** [comment]
Merged, thanks.

**Pull request merged**